docs/removed_features.rst

```diff
@@ -0,0 +1,23 @@
+Removed Features
+================
+
+This page aims at providing a short knowledge base designed to track dropped and potentially useful features for future development.
+To that end, the following table's columns constitute:
+
+* the name of the feature; it will also be used to tag last commit holding the feature.
+* a short description; the main purpose is to have the feature as greppable as possible.
+* removal PR number(s) related to their removal from the codebase.
+
+.. list-table:: Removed Features
+   :widths: 25 60 15
+   :header-rows: 1
+   :class: fixed-table
+
+   * - Name / Tag
+     - Short description
+     - Removal PR(s)
+   * - ni_pci_6229
+     - This related to a circa 2007 project implementing hard real time dynamic clamp under RTAI linux.
```

Comment:
@nrnhines took the liberty of borrowing your comment from #1399 and added the name of the card. Would this short description suffice?